docs/docs/building-a-blog.md

```diff
@@ -2,7 +2,63 @@
 title: Building a blog
 ---
 
-This is a stub. Help our community expand it.
 
-Please use the [Gatsby Style Guide](/docs/gatsby-style-guide/) to ensure your
-pull request gets accepted.
+### Building a blog
+
+Using gatsby-starter-blog you will build a full-featured blog. 
+
+### Prerequisites
+
+You must know how to set up a basic Gatsby project based on a starter. If you need to, check the [main tutorial](https://www.gatsbyjs.org/tutorial).
```

Comment:
The wording here could be just a little clearer "For instructions on how to setup a basic Gatsby project based on a starter, check the..."

Reply:
Changed the wording. 
Also I've changed the link to 'Using starters' section.